### ISSUE TITLE
Introduce Duration - element

### ISSUE THREAD

**Issue by geirmv** [emrex-eu/elmo-schemas#91]
**Meeting 28 June (Tor, Mirko, Jan-Joost, Geir)**

With the introduction of OOTS and support for education processes within Single Digital Gateway, there is a need to provide duration of studies to the ELMO schema.

We propose to add a new element LearnerOpportunityInstance.duration, with a type of duration - restricted to days, weeks, months, years.

```
      <xs:element name="duration" minOccurs="0">
        <xs:annotation>
          <xs:documentation>
           This describes the normal duration in time for an education.

            Here, issuers of ELMO can describe types of duration. The schema provides a predefined list of duration
          </xs:documentation>
        </xs:annotation>
        <xs:simpleType>
          <xs:restriction base="xs:token">
            <xs:enumeration value="Day">
              <xs:annotation>
                <xs:documentation>
                  Duration in number of days
                </xs:documentation>
              </xs:annotation>
            </xs:enumeration>
            <xs:enumeration value="Week">
              <xs:annotation>
                <xs:documentation>
                  Duration in number of weeks
                </xs:documentation>
              </xs:annotation>
            </xs:enumeration>
            <xs:enumeration value="Month">
              <xs:annotation>
                <xs:documentation>
                  Duration in number of months
                </xs:documentation>
              </xs:annotation>
            </xs:enumeration>
            <xs:enumeration value="Year">
              <xs:annotation>
                <xs:documentation>
                  Duration in number of years
                </xs:documentation>
              </xs:annotation>
            </xs:enumeration>

          </xs:restriction>
        </xs:simpleType>
      </xs:element>
```

**Comment by geirmv** [comment]
Propose that we use the standard xml type "duration" for this element.

**Comment by mathjert** [comment]
Solved in ELMO 2.1 https://github.com/emrex-eu/elmo-schemas/pull/97
